client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationIndicatorsColumn/SelfInvestigationIcon/SelfInvestigationIcon.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { Popover, Tooltip, ClickAwayListener, IconButton , Typography } from '@material-ui/core';
+
+import useStyles from './selfInvestigationIconStyles';
+import useSelfInvestigationIcon from './useSelfInvestigationIcon';
+import SelfInvestigationPopover from './Popover/SelfInvestigationPopover';
+
+const contactTitle = 'טופס תחקור עצמי';
+interface Props {
+    status: number;
+    date: Date;
+}
+
+const SelfInvestigationIcon = (props: Props) => {
+	const {status , date} = props;
+
+	const [isTooltipOpen, setIsTooltipOpen] = useState<boolean>(false);
+	const [anchorEl, setAnchorEl] = React.useState<HTMLButtonElement | null>(null);
+
+	const classes = useStyles();
+	const handleTooltipClose = (e : React.MouseEvent<Document, MouseEvent>) => {
+		e.stopPropagation();
+		setIsTooltipOpen(false);
+		setAnchorEl(null);
+	}
+	const handleButtonClick = (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => {
+		e.stopPropagation();
+		setAnchorEl(e.currentTarget)
+		setIsTooltipOpen((isOpen) => !isOpen);
+	}
+	const { getIconByStatus } = useSelfInvestigationIcon({status});
+
+	return (
+        <>
+            <ClickAwayListener onClickAway={handleTooltipClose}>
+                <Tooltip title={contactTitle} arrow placement={'top'}>
+                    <IconButton style={{padding : 0}} onClick={handleButtonClick}>{getIconByStatus()}</IconButton>
+                </Tooltip>
```

Comment:
i think theres a common component for tooltips w/ click listeners 🤔

Reply:
from what I can see its usually in the icon + tooltip thing is in commons/icons, but I think that since here the icon is dependent on status and that it's the only place that this icon is in use that it should be managed under the SelfInvestigationIcon folder 
I cant seem to find a generic icon/tooltip component though :/

Reply:
the comp name is ClickableTooltip and different icons can be passed conditionally 🙃

Reply:
![image](https://user-images.githubusercontent.com/68457306/103782242-e0d2b880-503f-11eb-81c0-12a6290a8013.png)

looks like the icon is pretty set to icons from the table (only offering primary and disabled color options for example) and including things like light tooltip/special popper instead of the popover i'm using now :/, 

this was actually originally how I implemented it but the field requested to have both a tooltip and a popover, so I separated them

Reply:
i would consider adding an optional color prop and make condition complex;
`const color = props.color || (value ? 'primary' : 'disabled')`